app/models/auction.rb

```diff
@@ -32,12 +32,12 @@ def lowest_amount
   end
 
   def start_price_equal_to_or_less_than_max_if_not_contracting_officer
-    if user && !user.contracting_officer? && start_price > MAX_START_PRICE
+    if user && !user.contracting_officer? && start_price > MICROPURCHASE_THRESHOLD
```

Comment:
Not really specific to this feature, but are we not allowing non-CO users to create/save auctions greater than the micropurchase threshold? I thought we were just preventing them from publishing them.